package.json

```diff
@@ -43,29 +43,16 @@
     "upupdowndownleftrightleftrightbastart"
   ],
   "dependencies": {
-    "@babel/core": "^7.20.12",
```

Comment:
I guess CLI has requirements for "@babel/core" in license and transpile js files.
Can we remove this?

Reply:
Yes, the "babel-preset-enact" repo has "@babel/core", and "cli" include this preset(that module is still included in the "cli" repo).
`enact license/transpile` command also passed in cli automation tests